### ISSUE TITLE
Speed up building of doxygen by unzipping MathJax once

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-core#1807]
See https://github.com/opensim-org/opensim-core/pull/1796#issuecomment-311573847.

**Comment by chrisdembia** [comment]
There also seems to be a bug with how I set up the CMake targets that causes MathJax to be unzipped *every* time you build doxygen (rather than only the first time you build doxygen).

**Comment by chrisdembia** [comment]
Okay, so the scenario I just experienced where the unzipping occurred for repeated builds of doxygen was because I used an old build directory where <build-dir>/MathJax.zip was actually MathJax 2.5, not 2.7. @tkuchida are you using an old build directory? If so, could you delete <build-dir>/MathJax.zip, re-run CMake, and build doxygen twice?

**Comment by tkuchida** [comment]
> could you delete /MathJax.zip, re-run CMake, and build doxygen twice?

Yeah, the old MathJax.zip file was the problem. Thanks! I'll let you decide whether to leave this issue open (MathJax.zip is still unzipped twice, but only on the first doxygen build).

**Comment by chrisdembia** [comment]
Thanks for trying that.

**Comment by chrisdembia** [comment]
I started to address this but then I realized that you would end up with *two* MathJax folders in the html directories, `MathJax-2.5-latest` and `MathJax-2.7.0`, which I think is an even worse outcome. We could detect if a 2.5 ZIP is already in the build directory, and then delete the html directories, but I think the scale of the problem doesn't warrant such a solution. I think it's fine to just request that members of the dev team delete `MathJax.zip` the next time they find themselves building doxygen.
